### PR TITLE
SEO-176113-Angular-Missing-H1

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on the missing H1

Regards,
Beryl.

### CODE REVIEW

File: angular/PdfViewer/Text-Markup-Annotation.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
-title: Text markup annotation of PDF viewer component.
-description: Text markup annotation of PDF viewer component.
+title: Text markup annotation | PDF viewer component | Syncfusion
```
Comment (Yvone-Atieno) [inline angular/PdfViewer/Text-Markup-Annotation.md:2]: Should include platform name

Comment (berylouma) [reply angular/PdfViewer/Text-Markup-Annotation.md]: Added

File: angular/PdfViewer/Text-Markup-Annotation.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
 ---
-title: Text markup annotation of PDF viewer component.
-description: Text markup annotation of PDF viewer component.
+title: Text markup annotation in Angular PDF viewer component | Syncfusion
```
Comment (Yvone-Atieno) [inline angular/PdfViewer/Text-Markup-Annotation.md:2]: Component, C should be in caps also in description

File: angular/PdfViewer/Text-Markup-Annotation.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 ---
 
-## Text Markup Annotation
+# Text Markup Annotation in Angular PDF Viewer
 
 PDF viewer control supports adding text markup annotations in the PDF documents. The component also renders the existing text markup annotations from the PDF document when the document is loaded in it.
 
 The following screenshot shows PDF viewer with the PDF document containing text markup annotations.
-![](Text-Markup-Annotation_images/Text_Markup_Annotations_img1.png)
+![Adding text markup in Angular PDF Viewer](Text-Markup-Annotation_images/Text_Markup_Annotations_img1.png)
```
Comment (Yvone-Atieno) [inline angular/PdfViewer/Text-Markup-Annotation.md:15]: It should be Text markup annotations in Angular PDF Viewer

File: angular/PdfViewer/Text-Markup-Annotation.md [hunk 4]
```diff
@@ -50,21 +50,21 @@ export class DefaultComponent {
 
 The colorpicker control is provided in the text markup annotation tools to select the desired color for the text markup annotation to be added in the document.
 
-![](Text-Markup-Annotation_images/Text_Markup_Annotations_img2.png)
+![Adding notes in Angular PDF Viewer](Text-Markup-Annotation_images/Text_Markup_Annotations_img2.png)
```
Comment (Yvone-Atieno) [inline angular/PdfViewer/Text-Markup-Annotation.md:53]: Adding text markup annotations in Angular PDF Viewer

File: angular/PdfViewer/Text-Markup-Annotation.md [hunk 5]
```diff
@@ -50,21 +50,21 @@ export class DefaultComponent {
...
 
 **Adding notes to the text markup annotation**
 
 The notes can be added to the text markup annotations using the context menu provided in the PDF viewer component.
 
 The below screenshot shows that the notes have been added to a text markup annotation in the PDF document.
-![](Text-Markup-Annotation_images/Text_Markup_Annotations_img3.png)
+![Editing the text markup in Angular PDF Viewer](Text-Markup-Annotation_images/Text_Markup_Annotations_img3.png)
```
Comment (Yvone-Atieno) [inline angular/PdfViewer/Text-Markup-Annotation.md:60]: Adding notes to text markup annotation in Angular PDF Viewer

File: angular/PdfViewer/Text-Markup-Annotation.md [hunk 6]
```diff
@@ -50,21 +50,21 @@ export class DefaultComponent {
...
 
 **Editing the text markup annotation**
 
 The properties of the text markup annotations in the PDF document can be edited using the properties window provided in the PDF viewer component. The color, opacity, author and subject of the text markup annotation can be modified. The properties of the text markup annotation can also be locked using the locked checkbox in the properties window.
 
 The below screenshot shows the properties window used for editing the properties of a text markup annotation.
-![](Text-Markup-Annotation_images/Text_Markup_Annotations_img4.png)
+![Saving the text markup in Angular PDF Viewer](Text-Markup-Annotation_images/Text_Markup_Annotations_img4.png)
```
Comment (Yvone-Atieno) [inline angular/PdfViewer/Text-Markup-Annotation.md:67]: Editing the text markup annotation in Angular PDF Viewer

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: angular/PdfViewer/Text-Markup-Annotation.md (hunk 6 above)
Comment (berylouma) [reply angular/PdfViewer/Text-Markup-Annotation.md]: Updated

Comment (berylouma) [reply angular/PdfViewer/Text-Markup-Annotation.md]: Updated

File: angular/PdfViewer/Text-Markup-Annotation.md (hunk 5 above)
Comment (berylouma) [reply angular/PdfViewer/Text-Markup-Annotation.md]: Updated

File: angular/PdfViewer/Text-Markup-Annotation.md (hunk 4 above)
Comment (berylouma) [reply angular/PdfViewer/Text-Markup-Annotation.md]: Updated

File: angular/PdfViewer/Text-Markup-Annotation.md (hunk 3 above)
Comment (berylouma) [reply angular/PdfViewer/Text-Markup-Annotation.md]: Updated

File: angular/PdfViewer/Text-Markup-Annotation.md (hunk 2 above)
Comment (berylouma) [reply angular/PdfViewer/Text-Markup-Annotation.md]: Updated

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Pull request merged**